native/libffi/README.md

```diff
@@ -108,6 +108,7 @@ tested:
 | X86             | Solaris          | Oracle Solaris Studio C |
 | X86             | Windows/Cygwin   | GCC                     |
 | X86             | Windows/MingW    | GCC                     |
+| X86-64          | DragonFly        | GCC                     |
```

Comment:
Please don't modify the embedded copy of libffi it is a verbatim copy from upstream.

Reply:
My mistake. I'll fix it.